### PR TITLE
chore(docs): Update kfp_tekton_install.md with kfp-tekton v2 install instructions

### PR DESCRIPTION
**Pull request by Tomcli**
**Which issue is resolved by this Pull Request:** 
Resolves #

**Description of your changes:**

**Environment tested:**

* Python Version (use `python --version`):
* Tekton Version (use `tkn version`):
* Kubernetes Version (use `kubectl version`):
* OS (e.g. from `/etc/os-release`):

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: guides/kfp_tekton_install.md [hunk 1]
```diff
@@ -103,6 +105,37 @@ To install the standalone Kubeflow Pipelines with Tekton, run the following step
...
+3. (GPU worker nodes only) If your Kubernetes cluster has a mixture of CPU and GPU worker nodes, it's recommended to disable the Tekton default affinity assistant so that Tekton won't schedule too many CPU workloads on the GPU nodes.
+    ```shell
+    kubectl patch cm feature-flags -n tekton-pipelines \
+      -p '{"data":{"disable-affinity-assistant": "true"}}'
+    ```
+
+Now, please use the [KFP V2 Python SDK](https://pypi.org/project/kfp/) to compile KFP-Tekton V2 pipelines because we are sharing the same pipeline spec starting from KFP V2.0.0. 
+
+```shell
+pip install "kfp>=2.0"
```
Comment (yhwang) [inline guides/kfp_tekton_install.md:136]: also `kfp-kubernetes` ?

Comment (Tomcli) [reply guides/kfp_tekton_install.md]: done

**Review by yhwang — APPROVED** [review]
/lgtm

**Pull request merged**